lxd/api_cluster.go

```diff
@@ -1071,6 +1069,7 @@ type internalClusterPostAcceptRequest struct {
 	API          int               `json:"api" yaml:"api"`
 	StoragePools []api.StoragePool `json:"storage_pools" yaml:"storage_pools"`
 	Networks     []api.Network     `json:"networks" yaml:"networks"`
+	Arch         int               `json:"arch" yaml:"arch"`
```

Comment:
I'd prefer if this also was `Architecture` and `architecture` even if it's less user visible.